src/views/ProfileSettingsPage.vue

```diff
@@ -14,6 +14,7 @@ import {
 const authUser = getAuth().currentUser!
 const authStore = useAuthStore()
 const icon = ref(authStore.icon)
+let uploadFile = new File([], '')
```

Comment:
uploadFileをリアクティブにして
下のonFileUploadToFirebaseメソッドでFileオブジェクト取得したときに更新するとか
（Fileオブジェクトの初期化の調べ方調べてません…）

```suggestion
const uploadFile = ref(new File([], ''))
```